reference/deploy-tokens.html.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Deploy Tokens
+layout: docs
+sitemap: false
+nav: firecracker
+---
+
+<div class="border border-violet-600 bg-violet-50 rounded-l p-4 my-4 text-base text-navy">
+Deploy tokens are an experimental feature. They may be less reliable than traditional API tokens. Use them with caution and visit our <a href="https://community.fly.io">community forum</a> for help.
+</div>
+
+Whether you're [deploying your app with GitHub Actions](/docs/app-guides/continuous-deployment-with-github-actions/) or running your own CD service, it's best to avoid configuring deployment infrastrcutre with all-powerful tokens. Deploy tokens can be used with `flyctl` to manage a single application and it's resources.
```

Comment:
```suggestion
Whether you're [deploying your app with GitHub Actions](/docs/app-guides/continuous-deployment-with-github-actions/) or running your own CD service, it's best to avoid configuring deployment infrastructure with all-powerful tokens. Deploy tokens can be used with `flyctl` to manage a single application and its resources.
```

Reply:
My spell checking plugin in VSCode doesn't work very well 😦